_includes/modern-appendix.md

```diff
@@ -843,6 +851,32 @@ The text used in the last param of this message varies wildly.
 
 Indicates that there was a problem with a mode parameter. Replaces various implementation-specific mode-specific numerics.
 
+{% numericheader RPL_HELPSTART %}
+
+    "<client> <subject> :<firt line of help section>"
+
+Indicates the start of a reply to a {% command HELP %} command to the client.
+The text used in the last param of this message may vary, and SHOULD be displayed as-is by IRC clients to their users; possibly emphasized as a the title of the help section.
+
+The `<subject>` MUST be the one requested by the client, up to casing; unless it would be an invalid parameter, in which case it MUST be `*`.
```

Comment:
Slight spelling fix and clarification of the `up to casing` part. If this new text is wrong, replace `but may be casefolded` with `case sensitive` instead.

The two other `up to casing` parts below should also be replaced with the new text.

```suggestion
    "<client> <subject> :<first line of help section>"

Indicates the start of a reply to a {% command HELP %} command.
The text used in the last param of this message may vary, and SHOULD be displayed as-is by IRC clients to their users; possibly emphasized as a the title of the help section.

The `<subject>` MUST be the one requested by the client, but may be casefolded; unless it would be an invalid parameter, in which case it MUST be `*`.
```